lesson_source/0036/pl.md

```diff
@@ -0,0 +1,98 @@
+1.16 Agile i Scrum
+=======================
+
+Manifest programowania zwinnego
+-------------------------------
+
+(ang. Manifesto for Agile Software Development)
+
+Odkrywamy nowe metody programowania dzięki praktyce w programowaniu
+i wspieraniu w nim innych.
+
+W wyniku naszej pracy, zaczęliśmy bardziej cenić:
+
+*Ludzi i interakcje od* procesów i narzędzi
+*Działające oprogramowanie* od szczegółowej dokumentacji
+*Współpracę z klientem* od negocjacji umów
+*Reagowanie na zmiany* od realizacji założonego planu.
+
+Oznacza to, że elementy wypisane po prawej są wartościowe,
+ale większą wartość mają dla nas te, które wypisano po lewej.
+
+1. Naszym najwyższym priorytetem jest zadowolić klienta poprzez wczesne i ciągłe dostarczanie wartościowego oprogramowania.
+2. Zmiany w wymaganiach, nawet późno w projekcie są mile widziane. Process Agile zaprzęga zmianę do przewagi we współzawodnictwie na korzyść klienta.
+3. Dostarczaj oprogramowanie często, w odstępach czasu od kilku tygodni do kilku miesięcy, preferując mniejsze odstępy czasowe.
+4. Ludzie biznesu i deweloperzy muszą pracować razem codziennie przez cały projekt.
+5. Buduj projekty w opraciu o zmotywowane indywidualności. Daj im środowisko i wsparcie, którego potrzebują i zaufaj im, że praca zostanie wykonana.
+6. Najskuteczniejszą i najwydajniejszą metodą przekazywania informacji do i w zespole deweloperów jest rozmowa w cztery oczy.
+7. Działające oprogramowanie jest podstawową miarą postępu.
+8. Procesy Agile promują trwały rozwój. Sponsorzy, deweloperzy i użytkownicy powinni być w stanie utrzymać ciągłe tempo nieskończenie.
```

Comment:
nie podoba mi się sformułowanie "ciągłe tempo nieskończenie" ale nie mam pomysłu jak inaczej to sformułować :C